src/components/LogRow/CollapsedRow/index.tsx

```diff
@@ -27,25 +30,23 @@ const CollapsedRow = forwardRef<any, CollapsedRowProps>((props, ref) => {
 
   const lineText =
     numCollapsed !== 1 ? `${numCollapsed} lines skipped` : "1 line skipped";
-  const disableExpandFive = 2 * SKIP_NUMBER > numCollapsed;
-
-  const [, startTransition] = useTransition();
-  const { sendEvent } = useLogWindowAnalytics();
 
   const expandFive = () => {
-    startTransition(() => {
-      expandLines([
-        [start, start + (SKIP_NUMBER - 1)],
-        [end - (SKIP_NUMBER - 1), end],
-      ]);
-    });
-    sendEvent({ name: "Expanded Lines", lineCount: 5, option: "Five" });
+    if (2 * SKIP_NUMBER < numCollapsed) {
+      startTransition(() =>
+        expandLines([
+          [start, start + (SKIP_NUMBER - 1)],
+          [end - (SKIP_NUMBER - 1), end],
+        ])
+      );
+      sendEvent({ name: "Expanded Lines", lineCount: 5, option: "Five" });
+    } else {
+      expandAll();
```

Comment:
I think it still makes sense to track this as Expanded Five since that's the button they clicked.